### PR TITLE
Add about pareksha

### PR DESCRIPTION
**Pull request by pareksha**

### CODE REVIEW

**Review by divyam3897 — CHANGES_REQUESTED** [review]
Contact infos not in different lines.

**Comment by pareksha** [comment]
Sir,
Kindly review the PR again. Changes have been made in the same PR.
Regards,
Pareksha

On 14 Oct 2017 11:50 am, "Divyam Madaan" <notifications@github.com> wrote:

*@divyam3897* requested changes on this pull request.

Contact infos not in different lines.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/pclubuiet/Hacktoberfest-2k17/pull/57#pullrequestreview-69385109>,
or mute the thread

File: submissions/parekshamanchanda/about.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# Experience
+No experience yet. 
+# Interests
+Python, Android. 
+# Contact Info
+Email: pareksha.manchanda@gmail.com  
+Github: pareksha  
+Facebook Profile: Pareksha Manchanda
+# Emoji
+My favorite emoji is :smile:
```
Comment (divyam3897) [inline submissions/parekshamanchanda/about.md]: full stop after the end of sentence.

**Review by divyam3897 — CHANGES_REQUESTED** [review]

**Comment by pareksha** [comment]
Changes have been made. Kindly, review PR again.

File: submissions/parekshamanchanda/about.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# About Me
+My name is Pareksha Manchanda. I am a CSE student at Panjab University, India. 
+# Experience
+No experience yet. 
+# Interests
+Python, Android. 
+# Contact Info
+Email: pareksha.manchanda@gmail.com
```
Comment (gurpreetsingh00885) [inline submissions/parekshamanchanda/about.md]: Wrong format for E-mail.

File: submissions/parekshamanchanda/about.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# About Me
+My name is Pareksha Manchanda. I am a CSE student at Panjab University, India. 
+# Experience
+No experience yet. 
+# Interests
+Python, Android. 
+# Contact Info
+Email: pareksha.manchanda@gmail.com  
+Github: pareksha
```
Comment (gurpreetsingh00885) [inline submissions/parekshamanchanda/about.md]: Wrong format for adding a link.

File: submissions/parekshamanchanda/about.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# About Me
+My name is Pareksha Manchanda. I am a CSE student at Panjab University, India. 
+# Experience
+No experience yet. 
+# Interests
+Python, Android. 
+# Contact Info
+Email: pareksha.manchanda@gmail.com  
+Github: pareksha  
+Facebook Profile: Pareksha Manchanda
```
Comment (gurpreetsingh00885) [inline submissions/parekshamanchanda/about.md]: same

**Review by gurpreetsingh00885 — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**